json-schemas/src/attachments.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably Attachment Event",
+  "type": "object",
+  "properties": {
+    "channelId": {
+      "type": "string",
+      "description": "The ID of the channel"
+    },
+    "channelName": {
+      "type": "string",
+      "description": "The name of the channel"
+    },
+    "qualifier": {
+      "type": "string",
+      "description": "The qualifier used in the channel spec"
+    },
+    "qualifierParam": {
```

Comment:
Would it be useful to also include the qualifier query (i.e. the `rewind=1` in `[?rewind=1]foo`)?

Reply:
@mschristensen did you intend to add this?